docs/doc_assets/css/styleguide.scss

```diff
@@ -1109,3 +1109,7 @@ $font-light: 300;
     @include external-link(external-link-alt, external-link-alt-hover);
   }
 }
+
+.heading-margin-alt {
+  margin-top: 3em;
+}
```

Comment:
Can you move this code block to https://github.com/lmarkzon/web-design-standards/blob/66f49a4c860a594b6780f455c5b578df567dbd4c/docs/doc_assets/css/styleguide.scss#L284 so it's within the "Styleguide content" section?